### ISSUE TITLE
No a11y support at all?

### ISSUE THREAD

**Issue by dominikwilkowski** [hernansartorio/react-nice-dates#3]
I'd love this lib if it was more accessible.
Like support for keyboard only and screen readers and some mild contrast issues here and there.

**Comment by weyert** [comment]
Yes, would you accept PRs to improve this? Mostly, this might be a good read which kind of keyboard navigations are expected for a date picker element: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html

**Comment by hernansartorio** [comment]
@dominikwilkowski thanks for creating the issue. Agreed, this should be the next priority.
 
@weyert sure that would be awesome! If not I might get to it in a few days, let me know. 

Thanks, that's a great resource, someone had already mentioned it [on Twitter](https://twitter.com/devongovett/status/1219733286303305728?s=20) and I was planning on adding it to the issue.
